src/main/java/stsjorbsmod/cards/wanderer/Patron.java

```diff
@@ -61,7 +54,7 @@ public void downgrade() {
             upgraded = false;
             name = originalName;
             initializeTitle();
-            baseDamage = DAMAGE;
+            baseDamage = DAMAGE + WrathField.wrathEffectCount.get(this);
```

Comment:
There might be other mod effects for this; probably the most reliable option would be to stick as close to how upgrading works as possible (ie, subtract `UPGRADE_PLUS_DAMAGE` rather than resetting)